recipes/servicestack/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "servicestack" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4874987b82e81a8eeb146b53df6b282f48f843aa1e4f17eecc24d723dc3c5247
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - dataclasses >=0.6
+    - dataclasses-json >=0.5.4
+    - marshmallow >=3.12.2
+    - python >=3.9
+    - requests >=2.25.1
+
+test:
+  requires:
```

Would be good to have an import test as well:
```suggestion
test:
  imports:
    - servicestack
  requires:
```

---

Included. I must have misunderstood the docs when it says ["List of Python modules or packages that will be imported in the test environment."](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports) to be an optional if your test code didn't already import your package which this does.